nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -1,4 +1,135 @@
+   unit: Mcal/MWh
+
+Linear Var Cost|Electricity|Nuclear:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/MWh
+
+Linear Var Cost|Electricity|Oil:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/MWh
+
+Operation and Maintenance Cost|Electricity|Coal:
+   description: Operation and maintenance cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: €/MWh
+
+Operation and Maintenance Cost|Electricity|Gas:
+   description: Operation and maintenance cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: €/MWh
+
+Operation and Maintenance Cost|Electricity|Nuclear:
+   description: Operation and maintenance cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: €/MWh
+
+Operation and Maintenance Cost|Electricity|Oil:
+   description: Operation and maintenance cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: €/MWh
+
+Shut-Down Cost|Electricity|Coal:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Shut-Down Cost|Electricity|Gas:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Shut-Down Cost|Electricity|Nuclear:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Shut-Down Cost|Electricity|Oil:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+Start-Up Cost|Electricity|Coal:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start-Up Cost|Electricity|Gas:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start-Up Cost|Electricity|Nuclear:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start-Up Cost|Electricity|Oil:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Variable Cost|Electricity|Biomass|Traditionnal:
```

Comment:
Can these variables be removed as there is now a clear separation into linear and fixed components of the operational costs?

Reply:
yes; I think I did create those variables long ago, before erik added the more detailed ones from tepes.

Reply:
@erikfilias I suggest we delete all variable cost| variables

Reply:
Ok, I got it.

Reply:
@erikfilias @danielhuppmann I was a bit quick in answering. ACtually we need to keep the variable cost variable as it is not the same thing than the linear var cost. linear var cost is if I understood well a component of a formulae used by comillas to compute what I call the variable cost. In my model the cost for thermal unit is quadratic, then we have 3 inputs: quadratic cost, variable cost and fixed cost. the operation cost is then quadratic cost * active power * active power + variable cost * active power + fixed cost. As we do not have any public data available for quadratic cost and fixed cost I had only defined the variable cost. 
Maybe this is equivalent to Operation and Maintenance cost (as the unit is €/MWh) but then I would propose to define it somow differently, eg
Operation and Maintenance Cost|Linear term|Electricity|Gas
or
Operation and Maintenance Cost|Electricity|Gas|Linear term
@erikfilias I thus think that the variables like Operational Cost|Linear Term|Electricity Generation|Coal: should be named a bit differently as they are not really costs (the unit does not include €), and it is somehow ambiguous...

Reply:
Yeah, I've just checked the differences. 
In openTEPES, we make a previous step in related to your variable cost.
E.g. we consider the following:
```
Operational Cost|Linear Term|Electricity Generation|...
Operation and Maintenance Cost|Electricity|...
Operational Cost|Constant Term|Electricity Generation|...
```
And, the next expressions are calculated:
```
**linear   term cost**   [€/GWh] = Operational Cost|Linear Term|Electricity Generation|... [Mcal/MWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Cost|Electricity|... [€/MWh] * 1e-3
**constant term cost**   [€/h]   = Operational Cost|Constant Term|Electricity Generation|... [Mcal/h] * Price|Primary Energy [€/Mcal]
```

The **linear   term cost** is equivalent to your **variable cost**, and **constant term cost** to your fixed cost, I think. 

Then, I think we could keep these  variables because are not equal. And, I agree we should to rename it.
The  `Operational Cost|Linear Term|Electricity Generation|...` is defined as the linear term (slope) of the heat rate straight line, and `Operational Cost|Constant Term|Electricity Generation|...`, as the constant term (intercept) of the heat rate straight line.
Maybe, I could propose new names based on their definitions, @sandrinecharousset, do you agree?

Reply:
@erikfilias yes would be more clear and avoid confusions :-) 
We also use the same kind of formulaes in our models but depending on the datasource, usually what you can find is the variable cost only..... Then in long-term models we use that one. (and in short term operational models we use the ones computed out of the formulas, but in that case we do have all the technical data)

Reply:
I suggest to express startup and shutdown costs in MEUR instead of Mcal.

Reply:
what is "MEUR"? if million, "million EUR", "mEUR" might be more intuitive...

Reply:
I suggest to keep M to be compatible with the International System of Units